docs/src/main/sphinx/release/release-462.md

```diff
@@ -0,0 +1,107 @@
+# Release 462 (16 Oct 2024)
+
+## General
+
+* Improve performance and memory consumption of the client protocol when
+  decoding data. ({issue}`23754`)
+
+
+
+
+## Database-based Resource Group Manager
+* Allow adding subgroups to groups without waiting for running queries to finish. ({issue}`23727`)
+* Fix query failures for queries routed to a group whose subgroup has been deleted. ({issue}`23727`)
+* Fix wrong resource group configuration being applied if the group is changed from a variable to fixed name or vice-versa. ({issue}`23727`)
+* Fix resource group updates not being observed immediately for groups that use variables. ({issue}`23727`)
+
+
+## Security
+
+
+## Web UI
+
+## JDBC driver
+
+* Improve performance and memory consumption when decoding data. ({issue}`23754`)
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+* Improve performance and memory consumption when decoding data. ({issue}`23754`)
+
+## Accumulo connector
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Exasol connector
+
+## Google Sheets connector
+
+## Hive connector
+
+## Hudi connector
+
+## Iceberg connector
+
+* Add support for read operations when using the Unity catalog as Iceberg REST
```

Comment:
@ebyhr I reworded your suggestion.. I hope this new version is a bit clearer. Also fyi @raunaqmorarka and @findinpath to confirm